exec/java-exec/src/main/java/org/apache/drill/exec/store/dfs/WorkspaceSchemaFactory.java

```diff
@@ -104,6 +107,9 @@ public WorkspaceSchemaFactory(DrillConfig drillConfig, FileSystemPlugin plugin,
       final FormatMatcher fallbackMatcher = new BasicFormatMatcher(formatPlugin,
           ImmutableList.of(Pattern.compile(".*")), ImmutableList.<MagicString>of());
       fileMatchers.add(fallbackMatcher);
+      dropFileMatchers = fileMatchers.subList(0, fileMatchers.size() - 1);
```

Comment:
The matchers are the same except for the pattern associated with defaultInputFormat. Because the pattern associated with defaultInputFormat is .* and we don't want to match and delete on this.